integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -451,23 +423,18 @@ void testDomainK8sEventsScalePastMax() {
    * Scale down and scale up the domain and verify that
    * DomainCompleted normal event is generated.
    */
-  @Order(7)
   @Test
   @DisplayName("Test domain completed event when domain is scaled.")
   void testScaleDomainAndVerifyCompletedEvent() {
-    try {
-      scaleDomainAndVerifyCompletedEvent(1, ScaleAction.scaleDown, true);
-      scaleDomainAndVerifyCompletedEvent(2, ScaleAction.scaleUp, true);
-    } finally {
-      scaleDomain(2);
-    }
+    createDomain(domainNamespace4, domainUid, pvName4, pvcName4);
+    scaleDomainAndVerifyCompletedEvent(1, ScaleAction.scaleDown, true, domainNamespace4);
+    scaleDomainAndVerifyCompletedEvent(2, ScaleAction.scaleUp, true, domainNamespace4);
```

Comment:
delete the domains when test is done so that it doesn't drain so much of cluster resource